doc/source/user_guide/groupby.rst

```diff
@@ -851,6 +851,13 @@ and that the transformed data contains no NAs.
 Window and resample operations
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. warning::
+
+    When using ``rolling()`` and an associated function the results are calculated with rolling sums. As a consequence
```

Comment:
I think this warning is more appropriate in `computation.rst` where a lot of the window documentation is

Reply:
Yes that is better, thanks